sources/packages/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -33,4 +40,49 @@ export class AESTSupportingUserController {
         } as ApplicationSupportingUsersDTO),
     );
   }
+
+  /**
+   * Get supporting user formName and the form data
+   * with the supportingUserId
+   * @param supportingUserId supporting user id.
+   * @param applicationId application id.
+   * @returns supporting user form data and details.
+   */
+  @Get(":supportingUserId/application/:applicationId")
+  async getSupportingUserFormDetails(
+    @Param("supportingUserId") supportingUserId: number,
+    @Param("applicationId") applicationId: number,
+  ): Promise<SupportingUserFormData> {
+    const supportingUserForms =
+      await this.applicationService.getSupportingUserFormName(applicationId);
```

Comment:
Why do we need a second method to do it? Is it not possible to bring this info altogether while executing the getSupportingUsersDetails?

Reply:
yup, it can be done. Thanks for that. I did this in 2 different parts first populating the form and then getting form data, didn't think of merging it. ya. ..updating, i liked it

Reply:
Thanks again for doing this change. Already saw it, looks good to me 👍